pygbe/main.py

```diff
@@ -355,14 +367,27 @@ def main(argv=sys.argv, log_output=True, return_output_fname=False,
     print('-'*30)
     print('Total setup time   : {}s'.format(setup_time))
 
-    tic = time.time()
+
+    #   Check if there is a complex dielectric
+    complexDiel = 0
+    for f in field_array:
```

Comment:
For some reason this doesn't work. It complains saying that there is a reference before assignment in this part:
```
    phi = numpy.zeros(param.Neq, type(f.E))
UnboundLocalError: local variable 'f' referenced before assignment
```